packages/binding-file/src/file-client.ts

```diff
@@ -24,40 +24,48 @@ import path = require("path");
 
 const { debug, warn } = createLoggers("binding-file", "file-client");
 
+/**
+ * Used to determine the Content-Type of a file from the extension in its
+ * {@link filePath} if no explicit Content-Type is defined.
+ *
+ * @param filepath The file paththe Content-Type is determined for.
```

Comment:
the comment is somehow broken

Reply:
Thank you :) Should be fixed by 907b429a5e4400de62a5999e59a24c18850e62a4